.github/workflows/e2e-test-backend.yml

```diff
@@ -28,7 +28,10 @@ jobs:
           sleep 5
           docker logs docker_managerapi_1
 
+      - name: set hosts
+        working-directory: ./api/test/e2e
+        run: echo "127.0.0.1 www.test2.com" | sudo tee -a /etc/hosts
```

Comment:
why need to change local file?

Reply:
That's because we can't make https request like this:

```
curl https://127.0.0.1:9443  -H 'Host: www.test2.com'
```

the SSL of www.test2.com will not match by this way. so need to set host and request www.test2.com directly.

Reply:
`curl --resolve 'www.test2.com:9443:127.0.0.1'  https://www.test2.com:9443`

you can make a try in this way.  ^_^

Reply:
thanks. I will try for it. but it's an example, we are not use curl to test.

Reply:
ok, got it. we can write some comment first, then we can fix it in a better way.

Reply:
It works!